lambdas/nino-check/src/helpers/audit.ts

```diff
@@ -12,12 +10,21 @@ export const sendRequestSentEvent = async (
   nino: string,
   deviceInformation?: string
 ) => {
-  await sendAuditEvent(REQUEST_SENT, {
-    auditConfig,
-    session,
-    personIdentity,
-    nino,
-    deviceInformation,
+  await sendAuditEvent(REQUEST_SENT, auditConfig, session, {
```

Comment:
I actually like that you removed this abstraction in the abandon lambda and it doesn't exist in the issue credential lambda. Is there a reason you kept it for this one?

Reply:
I suppose if it a **simple** event such as `END event`

 `await sendAuditEvent(END, functionConfig.audit, session);` this is nice from the perspective of the handler

From the handlers perspective it just wants to send an event with some data, it should NOT care about formatting the data. The VC_ISSUED event is now **15 lines of code**

Whereas, some abstracted code it would be 

```
      await sendAuditEvent(functionConfig.audit, session,{
          personIdentity,
          ninoUser.nino,
          evidence: [getAuditEvidence(attempts, contraIndicators, vcEvidence)]});
```